feat: fcc-todo-app (#51367)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-localstorage-by-building-a-todo-app/65003986d17d1e1865b269c0.md

---
id: 65003986d17d1e1865b269c0
title: Step 55
challengeType: 0
dashedName: step-55
---

# --description--

Now you should save the task items to local storage when the user adds, updates, or removes a task.

> Inside the `addOrUpdateTask` function, use `setItem()` to save the tasks with a key of `data`, then pass in the `taskData` array as it's parameter. Make sure you stringify the `taskData`.

Reviewer: ```suggestion
Inside the `addOrUpdateTask` function, use `setItem()` to save the tasks with a key of `data`, then pass the `taskData` array as its parameter. Ensure that you stringify the `taskData`.
```
